readme.md

```
@@ -15,120 +15,346 @@
+el host. El detalle completo del inventario de servicios está en [
+`docs/architecture.md`](https://github.com/ArnauAregall/aregall-agenthub/blob/main/docs/architecture.md).
 
 ### **2.3. Descripción de alto nivel del proyecto y estructura de ficheros**
 
-> Representa la estructura del proyecto y explica brevemente el propósito de las carpetas principales, así como si obedece a algún patrón o arquitectura específica.
+El proyecto es un monorepo con tres módulos: `backend/` (API Spring Boot), `frontend/` (SPA React/Vite), y `e2e/` (tests
+Playwright). El backend sigue una arquitectura de vertical slices donde cada dominio de negocio es un módulo cerrado. El
+frontend espeja esa misma estructura por features.
+
+La estructura completa del proyecto y las decisiones de arquitectura están documentadas en [
+`docs/architecture.md`](https://github.com/ArnauAregall/aregall-agenthub/blob/main/docs/architecture.md).
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside link text

(MD039, no-space-in-links)